src/Forms/ConfigurationForm.cs

```diff
@@ -223,6 +235,9 @@ public List<string> GetAzureMigrateSourceAppliances()
             if (PhysicalCheckBox.Checked == true)
                 azureMigrateSourceAppliances.Add("physical");
 
+            if (VMwareCheckBox.Checked == false && HyperVCheckBox.Checked == false && PhysicalCheckBox.Checked == false)
+                azureMigrateSourceAppliances.Add("import");

```

Comment:
This logic is wrong as per latest mocks,
Just add import if import is checked.

We will handle the disable and enable of appliance checkboxes on the form.

Reply:
Would modify this section according to the recent mocks.